packages/slate-react/src/utils/clone-fragment.js

```diff
@@ -82,6 +84,16 @@ function cloneFragment(event, value, fragment = value.fragment) {
 
   attach.setAttribute('data-slate-fragment', encoded)
 
+  //  Creates value from only the selected blocks
+  //  Then gets plaintext for clipboard with proper linebreaks for BLOCK elements
+  //  Via Plain serializer
+  const valFromSelection = Value.create({
```

Comment:
I believe you can do `Value.create({document: fragment})` instead as fragment is a slate's document

Reply:
This is accurate. I'll make that change now. Thank you